bitacross-worker/core/tls-websocket-server/src/connection.rs

```diff
@@ -149,6 +149,13 @@ where
 				Err(e) => match e {
 					tungstenite::Error::ConnectionClosed => return Ok(true),
 					tungstenite::Error::AlreadyClosed => return Ok(true),
+					tungstenite::Error::Io(e) => match e.kind() {
+						io::ErrorKind::BrokenPipe => return Ok(true),
```

Comment:
We will close the connection in case of any error except `io::ErrorKind::WouldBlock`

Reply:
Yes please : )